mail/context.go

```diff
@@ -24,25 +26,52 @@ func (c *context) toFlatMap() map[string]interface{} {
 
 // Recipient variable
 type ctxRecipient struct {
-	Name   string
-	Email  string
-	Params map[string]interface{}
+	Name        string
+	Email       string
+	Subject     string
+	Attachments []string
+	Params      map[string]interface{}
 }
 
-func newRecipient(data map[string]interface{}) ctxRecipient {
-	r := ctxRecipient{Params: keysToLower(data)}
+func newRecipient(data map[string]interface{}) (*ctxRecipient, error) {
+	r := &ctxRecipient{Params: keysToLower(data)}
 	r.Email, _ = r.Params["email"].(string)
 	r.Name, _ = r.Params["name"].(string)
+	r.Subject, _ = r.Params["subject"].(string)
+	if att, ok := r.Params["attachments"].([]interface{}); ok {
+		a := make([]string, len(att))
```

Comment:
You can use `ToStringSliceE` from `github.com/spf13/cast` here.

https://godoc.org/github.com/spf13/cast#ToStringSliceE